src/rebuild/srv.c

```diff
@@ -1875,6 +1875,20 @@ rebuild_tgt_prepare(crt_rpc_t *rpc, struct rebuild_tgt_pool_tracker **p_rpt)
 		}
 	}
 
+	if (pool->sp_iv_ns) {
+		/* XXX snapshot will not be allowed to be changed during
+		 * rebuild, let's invalidate local snapshot cache before
```

Comment:
You mean don't allow creating snapshot when there is a ongoing rebuild? I don't think it's reasonable and necessary.

Reply:
I thought we discussed this, and thought this is probably easier way.

Reply:
I don't think that'll be easier, you still need some sort of serialization to avoid race.